packages/amplify_datastore/ios/Classes/CognitoPlugin.swift

```diff
@@ -24,45 +24,24 @@ public class CognitoPlugin: AuthCategoryPlugin {
     
     public func configure(using configuration: Any?) throws {}
     
-    public func reset() async {
-        // TODO: reset plugin -- used for testing
```

Comment:
Q: should this TODO be left?

Reply:
Discussed offline, the [previous implementation](https://github.com/aws-amplify/amplify-flutter/blob/main/packages/amplify_datastore/ios/Classes/CognitoPlugin.swift#L48-L50) of this function was essentially a no-op and we're choosing to omit now.